packages/data-provider/README.md

```diff
@@ -0,0 +1,173 @@
+# Lyric (Data Provider)
+
+Package published on npm as https://www.npmjs.com/@overture-stack/lyric
+
+This package is part of Lyric mono repo, for more details go to [main page](../../README.md)
```

Comment:
Since this readme is the page we expect to see on npmjs once this is published, we should treat it like the primary information source for users discovering our package.

There are two main exports from this mono repo, this package and the server. This is the primary package people should import from npm, and the server is the containerized versoin that people will get the image of off of ghcr.io

What this means is we don't need this message here, we instead need a description of what this package does and why you would use it.

The first information up here should be what dependencies are needed to use this package and then a section that tells a developer how to use incorporate it into their own express server.

Developer instructions for how to install/edit the code content should be last in the readme or moved to a separate document with instructions for code contributors to get started.

Reply:
README file update with Installation and Usage to incorporate it into a Express Server.

Developer instructions have been moved on to a separate folder `data-provider/docs/`